src/stats/hist.jl

```diff
@@ -187,5 +190,8 @@ function Makie.plot!(plot::Hist)
         bp[1].val = points[]
         bp.width = w
     end
+    onany(plot.normalization, plot.scale_to, plot.weights) do _, _, _
```

Comment:
```suggestion
    onany(plot, plot.normalization, plot.scale_to, plot.weights) do _, _, _
```
This method should be used so that the callback is deregistered on close